### PR TITLE
#800 - Add MFIP Orientation to stand alone script and DAIL functionality

### PR DESCRIPTION
**Pull request by C-Love**
2 new scripts 
NOTES - MFIP Orientation
DAIL - MFIP Orientation

Update to DAIL Scrubber to access the new script.

Includes comments added to NOTES - Interview. 

Another pull will be needed for:
 - Update Complete List of Scripts
 - Update NOTES ChangeLog
 - Update FuncLib Changelog
 - retire DAIL - Financial Orientation Missing

### CODE REVIEW

**Comment by C-Love** [comment]
@IlseFerris - there are some questions in the code - let me know if those need clarification.

File: notes/mfip-orientation.vbs [hunk 1]
```diff
@@ -0,0 +1,957 @@
...
+		If minor_caregiver_on_case > 0 Then
+			Do
+				err_msg = ""
+				dlg_len = 210
+				If minor_caregiver_on_case = 2 Then dlg_len = 290
+
+				Dialog1 = ""
+				BeginDialog Dialog1, 0, 0, 551, dlg_len, "Assess for Caregiver MFIP Orientation Requirement"
+				  Text 10, 15, 200, 10, "Which Family Cash Program is this Application for? " & family_cash_program
+				  Text 10, 25, 500, 20, "Notes on Program Selection: " & famliy_cash_notes
```
Comment (IlseFerris) [inline notes/mfip-orientation.vbs:224]: family_cash_notes: This doesn't export to case note.

File: notes/mfip-orientation.vbs [hunk 2]
```diff
@@ -0,0 +1,957 @@
...
+'QUESTION - should we check to ensure MFIP is active or pending, or at least Cash is pending?
+
+'We are not using the function here because we do not need to select the members to look at.
+'The function has the ability to select which members of the household are caregivers in the next dialog.
+Call navigate_to_MAXIS_screen_review_PRIV("STAT", "MEMB", is_this_priv)			'navigating to stat memb to gather the list of all ref number on the case
+If is_this_priv = True Then Call script_end_procedure("It appears that this case is PRIVILEGED and you do not currently have access to it. Check the Case Number and, if needed, request access. Run the script agian once you confirm you have access to the case. The script will now end.")
+EMWriteScreen "01", 20, 76
+transmit
+
+EMReadScreen case_pw_county, 2, 21, 19											'check to see if the case is in another county
```
Comment (IlseFerris) [inline notes/mfip-orientation.vbs:824]: MEMB reads the case_pw_county at 2, 29, 23

**Review by IlseFerris — CHANGES_REQUESTED** [review]
I made notes on NOTES-MFIP ORIENTATION. I'm assuming they also apply to the DAIL script as well. 

General: I've added a new case note task to CLOSING ISSUE DOCUMENTATION after your pull request was sent.

**Comment by C-Love** [comment]
Updates made to the script based on review.

File: notes/mfip-orientation.vbs (hunk 2 above)
Comment (C-Love) [reply notes/mfip-orientation.vbs]: I think this is 21, 23 - that is what I changed it to.

File: notes/mfip-orientation.vbs (hunk 1 above)
Comment (C-Love) [reply notes/mfip-orientation.vbs:224]: UGH - yes - this became a whole thing - should be done now.

**Review by IlseFerris — COMMENTED** [review]
I approve thee!

**Pull request merged**